services/usage.rst

```diff
@@ -310,9 +310,10 @@ GitHub
 PFS project uses `Subaru-PFS organization at GitHub <https://github.com/Subaru-PFS>`_. 
 In 'Subaru-PFS' organization at GitHub, 
 there is one (mostly private) `team per institute <https://github.com/orgs/Subaru-PFS/teams>`_
-where the members have the admin privilege.
-You will need to be in one team of this organization 
-for push access to repositories. 
+where the members have the admin privilege to repositories owned.
+You will need to be in this organization (by invitation from the organization), 
```

Comment:
I think you can leave the old words to explain more clearly what you mean by admin privilege..
For instance,
> To push repositories in our organization, you need ...

And , just for clarification, you can change   "this" to "Subaru-PFS" .

Reply:
hrm. considering 'member of a team' is not a requirement, but 'member of Subaru-PFS organization' is a requirement, it seems this section need to be modified more to make clear...
how about newly pushed one?